### ISSUE TITLE
Exit inactivity mode whenever mouse is moved

### ISSUE THREAD

**Issue by Oscar-Rodrigues** [Digitopia/Gamult#63]
Right now, if app is in inactivity mode, you need a `touchDown()` (or `touchMoved()`, or `touchUp()`, but both of them require the `touchDown()`) to exit the `INACTIVE` state and go back to `ACTIVE`, which is counterintuitive.

**Comment by nunoh** [comment]
Can you rephrase that? I don't see how going from `INACTIVE` to `ACTIVE` after triggering a `touchDown()` is counter intuitive, but maybe I read you wrong.

**Comment by Oscar-Rodrigues** [comment]
My point exactly - inactivity is currently being handled on said methods,
so we should improve that
Nuno Hespanhol notifications@github.com escreveu em seg, 12/10/2015 às
22:06 :

**Comment by nunoh** [comment]
Sorry, I mistyped. I actually meant "counter intuitive". Updated the above post accordingly.

**Comment by Oscar-Rodrigues** [comment]
If we define inactivity as the lack of interaction, we should turn the
state to active as soon as there is some kind of interaction, such as
moving the mouse but not clicking
Nuno Hespanhol notifications@github.com escreveu em seg, 12/10/2015 às
22:26 :

**Comment by nunoh** [comment]
As validated with @Oscar-Rodrigues earlier today, closing this issue, at least for the moment, because it only applies when interacting with the app with the mouse, which is not the case in both the `Semibreve` and `Mobile` releases, which are the ones planned so far.
